### ISSUE TITLE
Settings save button isn't accessible by keyboard nav

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#4866]
When saving your changes on the settings page, I cannot reach the button with my keyboard only. This should be reachable without a mouse or mousepad.

**Comment by Floppy** [comment]
It does seem to be... perhaps something changed.
